addons/mail/static/src/components/message/message.scss

```diff
@@ -240,8 +244,12 @@
         }
     }
 
-    &.o-has-message-selection:not(.o-selected) {
-        opacity: 0.5;
+    .not-delivred {
+        color: $red;
+    }
+
+    a.not-delivred:hover, a.not-delivred:focus {
+        color: darken($red, 15%);
```

Comment:
Don't target node type, always target (just) class.
Note that this should probably be in `.o_Message_sendAgain` instead, avoid specificity.